ansible/cluster.yml

```diff
@@ -25,10 +25,51 @@
     - nodes
   sudo: yes
   roles:
-    - flannel
+    - { role: flannel, when: networking == 'flannel' }
   tags:
     - flannel
 
+# install opencontrail
+- hosts:
+    - masters[0]
+  sudo: yes
+  roles:
+    - { role: opencontrail-kmod, when: networking == 'opencontrail' }
+  tags:
+    - opencontrail-kmod
+
+- hosts:
+    - masters
+  gather_facts: yes
+  sudo: yes
+  tags:
+    - opencontrail
+    - opencontrail-gateway
```

Comment:
The intent is to be able to use variables such as "ansible_<interface-name>" in order for the nodes to know the ip address of the master.

Example:
```
server={%for host in groups['masters']%}{{ hostvars[host]['ansible_' + opencontrail_interface]['ipv4']['address'] }} {% endfor %}
```

My understanding is that gather_facts has been disabled at the start of the playbook. So this overrides that behavior when running these specific tags.